[XNNPACK] Enable XX kernels

digantdesai opened the pull request:
Summary: Enables copy, pad, fill etc. kernels in XNNPACK library. This shouldn't be too bad size implication.

Test Plan: CI

Differential Revision: D48915384

Skylion007 commented:
If we care so much about binary size, someone should really revisit https://github.com/pytorch/pytorch/issues/93955 as that would help quite a bit at reducing the size and is more feasible now that we have upgraded our min compiler version.

digantdesai committed:
[XNNPACK] Enable XX kernels

Summary: Enables copy, pad, fill etc. kernels in XNNPACK library. This shouldn't be too bad size implication. Per local testing,  after adding this, executor_runner size increases 2264 Bytes. 161299784 vs 161297520 (for x86 build non-opt build)

Test Plan: CI

Reviewed By: kirklandsign

Differential Revision: D48915384
